im-crawler/src/main/java/com/wipro/ats/bdre/imcrawler/frontier/WorkQueues.java

```diff
@@ -24,17 +25,13 @@
 import com.wipro.ats.bdre.imcrawler.url.WebURL;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.config.AutowireCapableBeanFactory;
-import org.springframework.context.ApplicationContext;
-import org.springframework.context.support.ClassPathXmlApplicationContext;
 
 import java.util.ArrayList;
 import java.util.List;
-
-
 /**
  * @author Yasser Ganjisaffar modified by AS294216
  */
-public class WorkQueues {
+public class WorkQueues extends MetadataAPIBase{
     private String dbName;
     //PersistenceManager manager = PMF.getInstance().getPersistenceManager();
     private final boolean resumable;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "resumable" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)